From: Tommy Webb <tommy@calyxinstitute.org>
Date: Wed, 9 Apr 2025 19:38:18 +0000
Subject: Increase wait before declaring HTTPS unavailable

Wait 10 seconds instead of 3 before showing the interstitial page that
declares a site "doesn't support a secure connection".

Issue: calyxos#2974
Change-Id: I40430dcfe757946e4a77acb37e92ab4e3e75ca55
---
 chrome/browser/ssl/https_upgrades_navigation_throttle.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/ssl/https_upgrades_navigation_throttle.cc b/chrome/browser/ssl/https_upgrades_navigation_throttle.cc
--- a/chrome/browser/ssl/https_upgrades_navigation_throttle.cc
+++ b/chrome/browser/ssl/https_upgrades_navigation_throttle.cc
@@ -34,7 +34,7 @@ namespace {
 
 // Time that the throttle will wait before canceling the upgraded navigation and
 // showing the HTTPS-First Mode interstitial.
-base::TimeDelta g_fallback_delay = base::Seconds(3);
+base::TimeDelta g_fallback_delay = base::Seconds(10);
 
 }  // namespace
 
-- 

